spk/aria2/Makefile

```diff
@@ -0,0 +1,24 @@
+SPK_NAME = aria2
+SPK_VERS = 1.31.0
+SPK_REV = 1
+SPK_ICON = src/aria2.png
```

Comment:
What I see on screen is `tmux` and `top`... If aria is included in a CLI multi-tool package, no need for an icon by the way.